app/views/layouts/_footer_content.html.slim

```diff
@@ -1,16 +1,16 @@
 .govuk-footer__section.govuk-grid-column-one-third
   ul.govuk-footer__list.govuk-footer__list--columns-1
-    li: a.govuk-footer__link href="/accessibility-statement"
-      | Accessibility statement
-    li: a.govuk-footer__link href="/settings/cookie-policy"
-      | Cookies
-    li: a.govuk-footer__link href="/privacy-policy"
-      | Privacy policy
+    li
+    = link_to "Accessibility statement", static_path('accessibility-statement'), class: "govuk-footer__link"
```

Comment:
Is there a difference if you use `govuk_link_to`? is it aware when it is used in a footer?

Reply:
tested, doesn't seem to make a difference with how it looks or works